packages/storybook-web-component/package.json

```diff
@@ -18,31 +18,22 @@
     "@babel/preset-env": "7.24.7",
     "@babel/preset-react": "7.24.7",
     "@babel/preset-typescript": "7.24.7",
-    "@etchteam/storybook-addon-status": "4.2.4",
+    "@etchteam/storybook-addon-status": "7.0.1",
     "@gemeente-denhaag/design-tokens-components": "0.2.3-alpha.359",
     "@mdx-js/react": "2.3.0",
     "@nl-design-system-unstable/amsterdam-design-tokens": "1.0.0-alpha.136",
     "@nl-design-system-unstable/documentation": "0.0.2-alpha.94",
     "@nl-design-system-unstable/rotterdam-design-tokens": "1.0.0-alpha.125",
-    "@storybook/addon-a11y": "7.6.4",
-    "@storybook/addon-docs": "7.6.4",
-    "@storybook/addon-links": "7.6.4",
-    "@storybook/addon-mdx-gfm": "7.6.4",
-    "@storybook/addon-viewport": "7.6.4",
-    "@storybook/addons": "7.6.4",
-    "@storybook/api": "7.6.4",
-    "@storybook/blocks": "7.6.4",
-    "@storybook/cli": "7.6.4",
-    "@storybook/components": "7.6.4",
-    "@storybook/core-events": "7.6.4",
+    "@storybook/addon-a11y": "9.1.5",
+    "@storybook/addon-docs": "9.1.5",
+    "@storybook/addon-links": "9.1.5",
+    "@storybook/cli": "9.1.5",
     "@storybook/mdx2-csf": "1.1.0",
     "@storybook/preset-scss": "1.0.3",
-    "@storybook/react": "7.6.4",
-    "@storybook/react-webpack5": "7.6.4",
-    "@storybook/theming": "7.6.4",
-    "@storybook/types": "7.6.4",
+    "@storybook/react-webpack5": "9.1.5",
     "@tabler/icons-react": "2.44.0",
     "@types/lodash": "4.14.202",
+    "@types/node": "24.3.1",
```

Comment:
Dit is niet in sync met de Node.js versie die we gebruiken, zoals ingesteld in `.nvmrc`. Ik zou hier een `22` versie verwachten.